marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserModulesCommand.java

```diff
@@ -188,7 +188,7 @@ else if (isEntityDir(dir, startPath.toAbsolutePath())) {
                                     InputStream inputStream = r.getInputStream();
                                     StringHandle handle = new StringHandle(IOUtils.toString(inputStream));
                                     inputStream.close();
-                                    entityDocMgr.write(URLEncoder.encode("/entities/" + r.getFilename(), "UTF-8"), meta, handle);
+                                    entityDocMgr.write("/entities/" + r.getFilename(), meta, handle);
```

Comment:
is this on purpose? doesn't seem related to the issue

Reply:
I accidentally checked the URLEncode in the other day. this is me undoing it.

Reply:
I should get this into develop for today's release and not on this branch.